server/datastore/mysql/software.go

```diff
@@ -2397,8 +2397,7 @@ func hostInstalledSoftware(ds *Datastore, ctx context.Context, hostID uint) ([]*
 			host_software.last_opened_at,
 			software.source AS software_source,
 			software.version AS version,
-			software.bundle_identifier AS bundle_identifier,
-			'installed' AS status
```

Comment:
This might have been committed by mistake as part of https://github.com/fleetdm/fleet/pull/28372

It is not safe to assume that entries in `host_software` have the "installed" status. Removing it doesn't seem to break any tests, so not sure why it was added.